### ISSUE TITLE
Come up with a canonical AppDev example app.

### ISSUE THREAD

**Issue by jordanlewis** [cockroachdb/appdev-issues#2]
We've talked about this a bit. This issue tracks coming up with the right idea.

Login?
Todo lists?
User management?
Wikifeedia?

Can be anything. We should think about what aspects of CockroachDB we should demonstrate, as well as what common developer-focused tasks we want to demonstrate as possible on CockroachDB.

Let's target either Django, Hibernate, Spring, or SQLAlchemy.

**Comment by rafiss** [comment]
@apantel is looking into Zanzibar, a project from Google https://www.usenix.org/conference/atc19/presentation/pang

**Comment by rafiss** [comment]
In order to complete this, can you push up the remaining code to get https://github.com/cockroachdb/django-todo/ working?

Once the PR is merged, we can close this. If needed, we can create a separate issue to write a quick blog post to publicize the example.
